packages/embark-whisper/src/index.js

```diff
@@ -4,10 +4,11 @@ import {joinPath, canonicalHost, defaultHost} from 'embark-utils';
 let Web3 = require('web3');
 const {parallel} = require('async');
 const {sendMessage, listenTo} = require('./js/communicationFunctions');
-const constants = require('../../constants');
 const {fromEvent} = require('rxjs');
 const {map, takeUntil} = require('rxjs/operators');
 
+const EMBARK_RESOURCE_ORIGIN = "http://embark";
```

Comment:
Would it be possible to pull this out in to `constants.json` and replace all occurrences in the codebase with that constant? We use this in a few places (building CORS comes to mind first) and would be nice to take the opportunity to consolidate it.